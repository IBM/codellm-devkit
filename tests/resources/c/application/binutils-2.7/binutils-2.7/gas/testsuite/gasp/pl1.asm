	
	.ALTERNATE

alloc 	MACRO val1,val2
	DB	val1
	DB 	val2
	ENDM

	alloc	"that's" 'show biz'
	alloc	0,1
	alloc	0 1
	alloc	0	1
	alloc	,1


	




