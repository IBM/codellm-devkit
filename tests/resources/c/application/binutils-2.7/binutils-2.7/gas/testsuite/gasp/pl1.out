!	
!	.ALTERNATE
!

!alloc 	MACRO val1,val2
!	DB	val1
!	DB 	val2
!	ENDM
!

!	alloc	"that's" 'show biz'
!	DB	"that's"
	.byte	116,104,97,116,39,115
!	DB 	"show biz"
	.byte	115,104,111,119,32,98,105,122
!	alloc	0,1
!	DB	0
	.byte	0
!	DB 	1
	.byte	1
!	alloc	0 1
!	DB	0
	.byte	0
!	DB 	1
	.byte	1
!	alloc	0	1
!	DB	0
	.byte	0
!	DB 	1
	.byte	1
!	alloc	,1
!	DB	
	.byte	
!	DB 	1
	.byte	1
!

!

!	
!

!

!

!

!